app/views/issues/show.html.erb

```diff
@@ -18,6 +18,11 @@
       <p id="header-intro"><%= @issue.description %></p>
     </div>
   </div>
+
```

Comment:
did hound hound you for this spacing being different than the candidate show spacing?